cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/sinks/NGSICKANSink.java

```diff
@@ -246,15 +248,15 @@ void persistBatch(NGSIBatch batch) throws Exception {
             for (NGSIEvent event : events) {
                 aggregator.aggregate(event);
             } // for
-
+            
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1360/commits/fed06a7b8b7685631e5b8fac86223d537a11c8ef